migrations/migrations.sql

```diff
@@ -391,6 +391,11 @@ do $$
     if not exists (select 1 from information_schema.columns where table_name = 'runs' and column_name = 'isPipelineTriggered') then
       alter table "runs" add column "isPipelineTriggered" BOOLEAN;
     end if;
+   
+    -- Adds runs propertyBag
+    if not exists (select 1 from information_schema.columns where table_name = 'runs' and column_name = 'propertyBag') then
+      alter table "runs" add column "propertyBag" TEXT;
```

Comment:
this will lock up the whole table. we cannot run this query directly. a better way would be to create a new table with the new column, add the data from old to new table and rename new table.  
@rageshkrishna im not merging this right now. pl take a look when you get a chance

Reply:
Just FYI we are not populating any default value so I don't think this should take much time.